src/restapi/fConnect.ts

```diff
@@ -111,21 +81,9 @@ export const fOauth2Complete: CallResponseHandler = async (req, res) => {
         token,
         role: me.role,
     };
-    try {
-        await ppClient.storeOauth2User(storedToken);
-    } catch (error) {
-        callResponse = newErrorCallResponseWithMessage('fOauth2Complete - Unable to store oauth2user: ' + error.message);
-        res.json(callResponse);
-        return;
-    }
+    await tryCallResponseWithMessage(ppClient.storeOauth2User(storedToken), 'fOauth2Complete - Unable to store oauth2user', res);
     const app = newApp(call);
-    try {
-        await app.createBotDMPost(dmText);
-    } catch (error) {
-        callResponse = newErrorCallResponseWithMessage('fOauth2Complete - Unable to create bot DM post: ' + error.message);
-        res.json(callResponse);
-        return;
-    }
-    callResponse = newOKCallResponse();
+    await tryCallResponseWithMessage(app.createBotDMPost(dmText), 'fOauth2Complete - Unable to create bot DM post', res);
+    const callResponse: AppCallResponse = newOKCallResponse();
```

Comment:
I genuinely feel that you can ignore this request, as there has already been a decent amount back-and-forth on the PR, and I don't want to bog down you with more requests. Though here is the suggestion:

I suggest we add a newline after every `tryCallResponseWithMessage` operation, mainly to improve readability.

```suggestion
    await tryCallResponseWithMessage(app.createBotDMPost(dmText), 'fOauth2Complete - Unable to create bot DM post', res);

    const callResponse: AppCallResponse = newOKCallResponse();
```

Reply:
Thanks Mick, I'm enjoying learning about codebase conventions. So since the emphasis would be on making important code readable, I should also add a space above if it looks cramped, and it's more of an art than a science, right? Feels like a silly question to ask, but when the scope is on spacing, I want to make sure I'm not making a mistake haha

Reply:
> So since the emphasis would be on making important code readable, I should also add a space above if it looks cramped

The convention is to "place newlines _after_ blocks". If it is a block that is preceeding the call to `tryCallResponseWithMessage` then there should be a newline. In the case of this project, a block would be any multi-line statement, whether that be a function call with its arguments are expanded across multiple lines, or an actual block such as an `if` block or `for` loop block.